

Hi,

Rainer reported that ctz-ch.c fails on several platforms.
This patch adds /* { dg-require-effective-target ctz } */
to the test which looks like it does the right thing.

Regtested on rv64gcv_zvl512, pushing as obvious.

Regards
 Robin

	PR tree-optimization/123192

gcc/testsuite/ChangeLog:

	* gcc.dg/tree-ssa/ctz-ch.c: Require ctz target.
---
 gcc/testsuite/gcc.dg/tree-ssa/ctz-ch.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gcc/testsuite/gcc.dg/tree-ssa/ctz-ch.c b/gcc/testsuite/gcc.dg/tree-ssa/ctz-ch.c
index 5d725979971..0842d44b4e1 100644
--- a/gcc/testsuite/gcc.dg/tree-ssa/ctz-ch.c
+++ b/gcc/testsuite/gcc.dg/tree-ssa/ctz-ch.c
@@ -1,4 +1,5 @@
 /* { dg-do compile } */
+/* { dg-require-effective-target ctz } */
 /* { dg-options "-O2 -fdump-tree-optimized" } */
 
 typedef unsigned long BITMAP_WORD;
-- 
2.51.1


